test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -101,11 +101,11 @@ class YacGateTest : public ::testing::Test {
     auto peer = makePeer("127.0.0.1", shared_model::crypto::PublicKey("111"));
     ledger_state = std::make_shared<iroha::LedgerState>(
         shared_model::interface::types::PeerList{std::move(peer)},
-        block->height(),
-        block->hash());
+        block->height() - 1,
+        block->prevHash());
   }
 
-  iroha::consensus::Round round{1, 1};
+  iroha::consensus::Round round{2, 1};
```

Comment:
```suggestion
  iroha::consensus::Round round{block->height(), 1};
```

Reply:
Block is not defined here yet. It might be better to use `round.block_round` for `block` variable initialization instead.